src/workerd/api/tests/url-test.js

```diff
@@ -9035,448 +9035,6 @@ export const urlPatternBasics = {
   },
 };
 
-export const urlPatternMdn1 = {
```

Comment:
Is it necessary to remove these existing tests? I know they might be duplicative of the web platform tests, but I still think there's value in them.

Reply:
I recommend removing them and relying on web-platform tests since MDN is not a good source for correctness.